managed-services/transactional-email/how-to/add-mx-record-to-your-domain.mdx

```diff
@@ -7,13 +7,13 @@ content:
   paragraph: This page explains how to add an MX record to your domain
 tags: transactional mx-record domain
 dates:
-  validation: 2023-05-15
+  validation: 2023-09-08
   posted: 2022-11-07
 categories:
   - managed-services
 ---
 
-This page explains how to set up an [MX](/managed-services/transactional-email/concepts/#mx-record) record to your domain to verify it.
+Setting up an MX record ensures that your emails are delivered reliably. This page explains how to set up Scaleway's [MX](/managed-services/transactional-email/concepts/#mx-record) record and how to set up a null MX record.
```

Comment:
we want to remove the mention of NULL MX. That is why we are providing an MX record. To remove the NULL MX, because some other mail server refuse email sent from a domain with a null MX